build.gradle

```diff
@@ -97,13 +95,10 @@ task diff << {
 					standardOutput = os
 				}
 
-				def output = os.toString()
+				def errorOutput = os.toString()
 
-				if (output?.trim()) {
-					println "\n"
-					println replicationFile.name
-					println "*******************************************************"
-					println output
+				if (!errorOutput.empty) {
+					throw new GradleScriptException("${replicationFiles} - ${replicationFile.name} \n", new Exception (errorOutput))
```

Comment:
Oh i see, you add it here :) nevermind on my previous comment.

However, this will through error on first problem found.  We need instead to save the errors in a list to output them all at the end and then throw the exception.